### PR TITLE
DE-325 Update API v1 and v2 03-28-2019

### PR DESCRIPTION
**Pull request by adducci**
Updates include:
- SearchFilter added to Search Properties
- Response removed from SystemUsersResetMfa endpoint
- List/ Create Provider Admin endpoints added

### CODE REVIEW

File: jcapiv2/docs/ProvidersApi.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# jcapiv2.ProvidersApi
```
Comment (lsi1verthorn) [inline jcapiv2/docs/ProvidersApi.md]: This ^^^ doesn't seem to follow the pattern of the others

Comment (adducci) [reply jcapiv2/docs/ProvidersApi.md]: I believe it's the same section vs member of the section difference, just generated in a different way

File: config_v1.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
   "packageName": "jcapiv1",
-  "packageVersion": "3.0.0"
+  "packageVersion": "3.1.0"
```
Comment (istevenson-JC) [inline config_v1.json]: Why is the package version 3.1.0 for Python, but 2.2.0 for Go and Ruby?

Comment (adducci) [reply config_v1.json]: It's based on the release version, but I'm not sure why/when they diverged.

**Pull request merged**